src/common/Icons.js

```diff
@@ -57,10 +57,11 @@ export type SpecificIconType = ComponentType<$Diff<IconProps<empty>, {| name: mi
 const makeIcon = <Glyphs: string>(
   iconSet: ComponentType<IconPropsBusted<Glyphs>>,
   name: Glyphs,
-): SpecificIconType => props => {
-  const FixedIcon = fixIconType<Glyphs>(iconSet);
-  return <FixedIcon name={name} {...props} />;
-};
+): SpecificIconType =>
+  function RNVIIcon(props) {
```

Comment:
This definition seems to thrash a bit: it gets reformatted more in the main upgrade commit, then reformatted partially back in the "Enable react/function-component-definition" commit.

I guess the latter commit changes `props => {` to `function RNVIIcon(props) {`. Maybe that causes the change? Seems like a bit of an annoying Prettier instability if so.

(but, to be clear, one of many -- not a regression to worry about)